src/internal/m365/collection/drive/helper_test.go

```diff
@@ -1543,46 +1547,31 @@ func toPath(elems ...string) string {
 	}
 }
 
-func fullPath(elems ...string) string {
+// produces the full path for the provided drive
+func (dd *deltaDrive) strPath(elems ...string) string {
 	return toPath(append(
 		[]string{
 			tenant,
 			path.OneDriveService.String(),
 			user,
 			path.FilesCategory.String(),
-			odConsts.DriveFolderPrefixBuilder(id(drivePfx)).String(),
+			odConsts.DriveFolderPrefixBuilder(dd.id).String(),
 		},
 		elems...)...)
 }
 
-func fullPathPath(t *testing.T, elems ...string) path.Path {
-	p, err := path.FromDataLayerPath(fullPath(elems...), false)
+func (dd *deltaDrive) fullPath(t *testing.T, elems ...string) path.Path {
+	p, err := path.FromDataLayerPath(dd.strPath(elems...), false)
 	require.NoError(t, err, clues.ToCore(err))
 
 	return p
 }
 
-func driveFullPath(driveID any, elems ...string) string {
+// produces a complete path prefix up to the drive root folder, and including any
+// elements passed in.
```

Comment:
nit: minor rewording for clarity
```suggestion
// produces a complete path prefix up to the drive root folder with any
// elements passed in appended to the generated prefix.
```